psi4/src/psi4/libpsio/toclen.cc

```diff
@@ -32,17 +32,6 @@
  */
 
 #include <cstdio>
-#ifdef _MSC_VER
-#include <io.h>
-#define SYSTEM_LSEEK ::_lseek
```

Comment:
Only possible trouble I see is here with `_lseek` vs `_lseeki64`. Docs say: `_lseek returns the offset, in bytes, of the new position from the beginning of the file. _lseeki64 returns the offset in a 64-bit integer.` Harmless?

Reply:
Eagle-eyed review! This is something I have not noticed until now.
`volseek.cc` used `_lseeki64` on Windows, while `toclen.cc` used `_lseek`.
Re-drafting this until I have had time to ponder the docs and the impact on the code.

Reply:
In `volseek.cc` the return value is never stored, only compared with -1, so 32/64 bit should not matter.
In `toclen.cc`, the return value is stored in an `auto` variable, which is then only compared with -1.
So I think it is harmless.